client/components/QuickStatsBar/QuickStatsBar.tsx

```diff
@@ -74,7 +84,9 @@ class QuickStatsBar extends Component {
           <div className="quick-stats-bar__stat">
             <SideEffectIcon className="quick-stats-bar__stat-icon" />{' '}
             <span>
-              {hasSideEffects.length ? 'some side-effects' : 'side-effect free'}
+              {!(hasSideEffects === false) && hasSideEffects.length
```

Comment:
Nit — this can simply be `hasSideEffects && hasSideEffects.length` perhaps?

Reply:
You are correct.